### PR TITLE
Adds ignore-rules section to axe

### PR DESCRIPTION
**Pull request by joeyciechanowicz**
Disables rules in axe using the `options.ignore` array. To enable the rules to be cross-compatible with HTML_CodeSniffer we add filtering based on the aXe ruleset.

Fixes https://github.com/pa11y/pa11y-ci/issues/145

### CODE REVIEW

File: lib/runners/htmlcs.js [hunk 1]
```diff
@@ -48,13 +48,15 @@ runner.run = async options => {
...
-				}
+		if (!options.rules.length || options.standard === 'Section508') {
+			return;
+		}
+
+		for (const rule of options.rules) {
+			if (window.HTMLCS_WCAG2AAA.sniffs.includes(rule)) {
+				window[`HTMLCS_${options.standard}`].sniffs[0].include.push(rule);
+			} else {
+				throw new Error(`${rule} is not a valid WCAG 2.0 rule`);
```
Comment (hollsk) [inline lib/runners/htmlcs.js:59]:
```suggestion
				throw new Error(`${rule} is not a valid WCAG 2.1 rule`);
```

File: lib/runners/htmlcs.js [hunk 2]
```diff
@@ -48,13 +48,15 @@ runner.run = async options => {
...
 	 */
 	function configureHtmlCodeSniffer() {
-		if (options.rules.length && options.standard !== 'Section508') {
-			for (const rule of options.rules) {
-				if (window.HTMLCS_WCAG2AAA.sniffs.includes(rule)) {
-					window[`HTMLCS_${options.standard}`].sniffs[0].include.push(rule);
-				} else {
-					throw new Error(`${rule} is not a valid WCAG 2.0 rule`);
-				}
+		if (!options.rules.length || options.standard === 'Section508') {
```
Comment (hollsk) [inline lib/runners/htmlcs.js:51]: we removed Section 508 as a ruleset option in the 6.0 release. Though can the option can still be passed through to pa11y from Pa11y-CI?

**Review by hollsk — COMMENTED** [review]
Joey you are a hero. A hero!!

File: lib/runners/htmlcs.js (hunk 2 above)
Comment (joeyciechanowicz) [reply lib/runners/htmlcs.js:51]: Hmmm I just did this to remove a lint warning. As we filter the rules below based on `HTMLCS_WCAG2AAA` anyway, I think we can drop this.

**Review by hollsk — APPROVED** [review]
✨

**Pull request merged**